frontend/src/components/pages/chat/components/messageList/MessageList.tsx

```diff
@@ -0,0 +1,159 @@
+import Box from '@material-ui/core/Box';
+import moment from 'moment';
+import React, { useEffect, useState } from 'react';
+import { Compose } from '../compose/Compose';
+import { Message } from '../message/Message';
+import './messageList.scss';
+
+const MY_USER_ID = 'apple';
+
+export const MessageList: React.FC = () => {
+  const [messages, setMessages] = useState<Array<any>>([]);
+
+  useEffect(() => {
+    getMessages();
+  }, []);
+
+  const getMessages = () => {
+    var tempMessages = [
+      {
+        id: 1,
+        author: 'apple',
+        message:
+          'Hello world! This is a long message that will hopefully get wrapped by our message bubble component! We will see how well it works.',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 2,
+        author: 'orange',
+        message: 'It looks like it wraps exactly as it is supposed to. Lets see what a reply looks like!',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 3,
+        author: 'orange',
+        message:
+          'Hello world! This is a long message that will hopefully get wrapped by our message bubble component! We will see how well it works.',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 4,
+        author: 'apple',
+        message: 'It looks like it wraps exactly as it is supposed to. Lets see what a reply looks like!',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 5,
+        author: 'apple',
+        message:
+          'Hello world! This is a long message that will hopefully get wrapped by our message bubble component! We will see how well it works.',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 6,
+        author: 'apple',
+        message: 'It looks like it wraps exactly as it is supposed to. Lets see what a reply looks like!',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 7,
+        author: 'orange',
+        message:
+          'Hello world! This is a long message that will hopefully get wrapped by our message bubble component! We will see how well it works.',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 8,
+        author: 'orange',
+        message: 'It looks like it wraps exactly as it is supposed to. Lets see what a reply looks like!',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 9,
+        author: 'apple',
+        message:
+          'Hello world! This is a long message that will hopefully get wrapped by our message bubble component! We will see how well it works.',
+        timestamp: new Date().getTime()
+      },
+      {
+        id: 10,
+        author: 'orange',
+        message: 'It looks like it wraps exactly as it is supposed to. Lets see what a reply looks like!',
+        timestamp: new Date().getTime()
+      }
+    ];
+    setMessages([...messages, ...tempMessages]);
+  };
+
+  const renderMessages = () => {
```

Comment:
This function looks overcomplicated, 
If you want to group messages I suggest to run reduce on messages firs to group them by array of arrays
And then loop over and add properties (first, last etc)